### PR TITLE
layout post of product

### PR DESCRIPTION
**Pull request by ghost**
## Description
layout post of product
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: resources/views/user/post/postOfProduct.blade.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="col-sm-12">
+		<h5 class="entry-title">
```
Comment (at-dungle) [inline resources/views/user/post/postOfProduct.blade.php]: chỉnh tab

File: resources/views/user/post/postOfProduct.blade.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+			<i class="fa fa-star"></i>
+			<i class="fa fa-star"></i>
+			<i class="fa fa-star-o"></i>
+			<i class="fa fa-star-o">
+			</i>&nbsp; <span>(5 votes)</span>
+		</div>
+		<div class="entry-excerpt"> 
+			Cras Pretium Arcu Ex.
+		</div>
+</div>
```
Comment (at-dungle) [inline resources/views/user/post/postOfProduct.blade.php]: xuống dòng

File: resources/views/user/post/postOfProduct.blade.php (hunk 1 above)
Comment (at-sonle) [reply resources/views/user/post/postOfProduct.blade.php]: done

**Comment by at-dungle** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**